.github/workflows/example.yml

```diff
@@ -0,0 +1,33 @@
+name: Run example
+on:
+  workflow_dispatch:
+    inputs:
+      example_paths:
+        type: string
+        required: true
+  workflow_call:
+    inputs:
+      example_paths:
+        type: string
+        required: true
+jobs:
+  run_example:
+    name: Run example
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        example_path: ${{ fromJSON(inputs.example_paths) }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: denoland/setup-deno@004814556e37c54a2f6e31384c9e18e983317366 # v1.1.0
+        with:
+          deno-version: v1.x
+      - name: Cache Deno dependencies
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/deno
+          key: ${{ runner.os }}-build-${{ hashFiles('lock.json', 'deps/**/*.ts') }}
+      - name: Setup Polkadot
+        uses: ./.github/actions/setup-polkadot
+      - run: deno task run test_ctx.ts deno task run ${{ matrix.example_path }}
```

Comment:
What do you think about adding a deno task `run:test_ctx` or similiar?

Reply:
I suppose we could, although it doesn't do much for us.

```
deno task run:test_ctx
```

vs.

```
deno task run test_ctx.ts
```

Reply:
Something like
```
deno task run:test_ctx ${{ matrix.example_path }}
```
Instead of
```
deno task run test_ctx.ts deno task run ${{ matrix.example_path }}
```

Reply:
Resolved by #304